extensions/reactive-datasource/runtime/src/main/java/io/quarkus/reactive/datasource/runtime/DataSourceReactiveRuntimeConfig.java

```diff
@@ -109,6 +109,13 @@
     @ConfigItem(defaultValue = "PT1S")
     public Duration reconnectInterval = Duration.ofSeconds(1L);
 
+    /**
+     * The hostname verification algorithm to use in case the server's identity should be checked.
+     * Should be HTTPS, LDAPS or an empty string.
+     */
+    @ConfigItem
+    public Optional<String> hostnameVerificationAlgorithm = Optional.empty();
```

Comment:
I was wondering if I should introduce an enum here? But all the other enums are typically already defined in Vertx. Here, the Vertx config is just a String, so we would need to define the enum ourselves in reactive-datasource.

Reply:
An enum wouldn't provide much, I think. The value depends on the type of client (e.g.g MySQL vs PostgreSQL).